components/mdx/Layout.tsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react'
```

Comment:
I think the file should be named something else since there's already a file called Layout.

Reply:
Renamed.